packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Migrate doc to terra-avatar
```

Comment:
```suggestion
  * Migrate doc from terra-avatar
```

Reply:
Shouldn't this also be in past tense? `Migrated`?